<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Repatriator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="repatriator.qrc">
    <normaloff>:/general/resource/bug.ico</normaloff>:/general/resource/bug.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="ImageDisplayWidget" name="displayWidget" native="true">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>271</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="controlsDock">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>C&amp;ontrols</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="sensitivityLabel">
       <property name="text">
        <string>S&amp;ensitivity: 100%</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="buddy">
        <cstring>sensitivitySlider</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sensitivitySlider">
       <property name="maximum">
        <number>4</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ShadowMinimap" name="shadowMinimap" native="true">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="labelAlignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&amp;Zoom</string>
         </property>
         <property name="buddy">
          <cstring>liftSliderZ</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="ShadowSlider" name="liftSliderZ">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&amp;Tilt</string>
         </property>
         <property name="buddy">
          <cstring>orbitSliderB</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="ShadowSlider" name="orbitSliderB">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&amp;Spin</string>
         </property>
         <property name="buddy">
          <cstring>orbitSliderA</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="ShadowSlider" name="orbitSliderA">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="snapshotButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="icon">
          <iconset resource="repatriator.qrc">
           <normaloff>:/general/resource/camera.png</normaloff>:/general/resource/camera.png</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+T</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoFocusEnabledCheckBox">
         <property name="text">
          <string>&amp;AutoFocus</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="filesDock">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>&amp;Downloadable Pictures</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="picturesList">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="movement">
        <enum>QListView::Static</enum>
       </property>
       <property name="viewMode">
        <enum>QListView::IconMode</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>646</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSelectAll"/>
    <addaction name="separator"/>
    <addaction name="actionDownload"/>
    <addaction name="actionDiscardSelectedFiles"/>
    <addaction name="separator"/>
    <addaction name="actionDownloadAllAndQuit"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Camera">
    <property name="title">
     <string>&amp;Camera</string>
    </property>
    <addaction name="actionTakeSnapshot"/>
   </widget>
   <widget class="QMenu" name="menuSe_ttings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionChangeDownloadFolder"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="menuSe_ttings"/>
   <addaction name="menu_Camera"/>
   <addaction name="viewMenu"/>
  </widget>
  <widget class="QDockWidget" name="bookmarksDock">
   <property name="windowTitle">
    <string>&amp;Bookmarks</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QListWidget" name="bookmarksList"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="goToBookmarkButton">
           <property name="text">
            <string>&amp;Go To</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editBookmarkButton">
           <property name="text">
            <string>&amp;Edit...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteBookmarkButton">
           <property name="text">
            <string>&amp;Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bookmarkHereButton">
           <property name="text">
            <string>&amp;Bookmark Here</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="newBookmarkButton">
           <property name="text">
            <string>&amp;New Bookmark...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="locationsDock">
   <property name="windowTitle">
    <string>&amp;Locations</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="locationsLayout"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionDownload">
   <property name="text">
    <string>&amp;Download Selected Pictures</string>
   </property>
   <property name="toolTip">
    <string>Downloads the selected pictures to your chosen download directory and then discards them from the server.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDiscardSelectedFiles">
   <property name="text">
    <string>D&amp;iscard Selected Pictures</string>
   </property>
   <property name="toolTip">
    <string>Deletes the selected pictures from the server.</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="actionDownloadAllAndQuit">
   <property name="text">
    <string>Download &amp;All and Quit</string>
   </property>
   <property name="toolTip">
    <string>Downloads all the pictures from the server to your chosen download directory and then deletes them from the server and then shuts down the application.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Shuts down the application.</string>
   </property>
  </action>
  <action name="actionTakeSnapshot">
   <property name="icon">
    <iconset resource="repatriator.qrc">
     <normaloff>:/general/resource/camera.png</normaloff>:/general/resource/camera.png</iconset>
   </property>
   <property name="text">
    <string>Take &amp;Snapshot</string>
   </property>
   <property name="toolTip">
    <string>Snap a detailed photo that you can download later.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionChangeDownloadFolder">
   <property name="text">
    <string>Change &amp;Download Destination Folder...</string>
   </property>
   <property name="toolTip">
    <string>Lets you choose a new location where you want pictures to be downloaded to.</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>&amp;Select All</string>
   </property>
   <property name="toolTip">
    <string>Highlight all of the downloadable pictures</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ShadowSlider</class>
   <extends>QSlider</extends>
   <header>ShadowSlider.h</header>
  </customwidget>
  <customwidget>
   <class>ShadowMinimap</class>
   <extends>QWidget</extends>
   <header>ShadowMinimap.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImageDisplayWidget</class>
   <extends>QWidget</extends>
   <header>ImageDisplayWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>snapshotButton</tabstop>
  <tabstop>picturesList</tabstop>
 </tabstops>
 <resources>
  <include location="repatriator.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionSelectAll</sender>
   <signal>triggered()</signal>
   <receiver>picturesList</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bookmarksList</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>goToBookmarkButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
